victor-mono: Move LICENSE.txt to fonts directory

afh opened the pull request:
###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

It seems cleaner to place Victor Mono's LICENSE.txt next to the fonts files with a font specific, rather than a generic filename. Otherwise the LICENSE.txt from Victor Mono gets symlinked into ~/.nix-profile, which seems less useful.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

afh committed:
victor-mono: Move LICENSE.txt to fonts directory

afh commented on pkgs/data/fonts/victor-mono/default.nix:
```diff
@@ -21,6 +21,7 @@ fetchzip {
 
     mv $out/OTF $out/share/fonts/opentype
     mv $out/TTF $out/share/fonts/truetype
+    mv $out/LICENSE.txt $out/share/fonts/$name-LICENSE.txt
```
Thanks for improving on this @Artturin 👌 

I think removing the `LICENSE.txt` altogether from `$out` is fine if the license declaration in [`pkgs/data/fonts/victor-mono/default.nix:33`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/victor-mono/default.nix#L33) is in accordance with [section 2 of the Victor Mono license](https://github.com/rubjo/victor-mono/blob/master/LICENSE#L56-L61):

> 2) Original or Modified Versions of the Font Software may be bundled,
redistributed and/or sold with any software, provided that each copy
contains the above copyright notice and this license. These can be
included either as stand-alone text files, human-readable headers or
in the appropriate machine-readable metadata fields within text or
binary files as long as those fields can be easily viewed by the user.

Artturin replied:
I think the following satisfies

> appropriate machine-readable metadata fields within text or
binary files as long as those fields can be easily viewed by the use

```json
artturin@nixos ~ % nix eval "nixpkgs#victor-mono.meta.license" --json | jq
{
  "deprecated": false,
  "free": true,
  "fullName": "SIL Open Font License 1.1",
  "redistributable": true,
  "shortName": "ofl",
  "spdxId": "OFL-1.1",
  "url": "https://spdx.org/licenses/OFL-1.1.html"
}
```

afh replied:
Great, let's remove the `LICENSE.txt` from `$out` then to keep `~/.nix-profile` clean and pristine 🙂 🧼🫧

Thanks for sharing the `nix eval` example, @Artturin. I learned something new about nix and find it quite useful 🙏 so went ahead and created simple zsh helper function named `nix/meta` for convenient nix derivation meta data inspection:
`nix eval nixpkgs#${1:-nix}.meta --json | jaq -r .${2}`

afh committed:
victor-mono: Remove LICENSE.txt

as "appropriate machine-readable metadata fields within text […]
can be easily viewed by the user", e.g.:
nix eval "nixpkgs#victor-mono.meta.license" --json | jq

afh commented:
Thanks for merging, @Artturin, much appreciated :pray:
